### PR TITLE
Update to LZ deployment Docs

### PR DESCRIPTION
**Pull request by cartyc**
Updating documentation on LZ deployment. 

This PR is aimed at improving the documentation around deploying with `kpt` and GitOps options. I am hoping to make things clearer as to what the different options do and when to use them.

Leaving this as a draft to collect more input and to test the new instructions around using OCI.

### CODE REVIEW

File: solutions/landing-zone/README.md [hunk 1]
```diff
@@ -102,21 +102,11 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
 
 1. Deploy Bootstrap
 
-    ```
-    arete create landing-zone-controller --region=northamerica-northeast1
-    ```
+    This bootstrap assume you have a [Config Controller](https://cloud.google.com/anthos-config-management/docs/concepts/config-controller-overview) instance provisioned already. If you do not you can follow either the [quickstart]https://cloud.google.com/anthos-config-management/docs/concepts/config-controller-overview) guide or the [detailed install guide](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/advanced-install.md) for advanced users.
```
Comment (fmichaelobrien) [inline solutions/landing-zone/README.md]: better, for a new user they would have stood up a kcc cluster and then had the instructions to setup another one in the solution.

I recommend for advanced users to directly to the advanced install and they will know to skip the arete step for cc cluster creation in the solution

File: solutions/landing-zone/README.md [hunk 2]
```diff
@@ -172,9 +162,22 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
...
+
+    When deploying resources `kpt` will wait for all resources to deploy before finishing, `kpt` also provides annotations such as [depends-on](https://kpt.dev/reference/annotations/depends-on/) that allow us to order the deployment of resources. 
+
+    GitOps allows us to connect our cluster to a "source of truth" and deploy from that. To do this we use [Anthos Config Management](https://cloud.google.com/anthos/config-management), which is pre-installed in Config Controller, and this allows us to target either a [Git Repository](https://cloud.google.com/anthos-config-management/docs/concepts/configs) or [OCI Artifact](https://cloud.google.com/anthos-config-management/docs/how-to/publish-config-registry) (Container). The benefit to this method is we now have an additional reconciliation process to ensure the deployed state matches our desired state or "source of truth". This also allows for improved automation and removes the need to run `kpt live apply` to deploy resources.
+
+    ### kpt
     
     ```
     kpt fn render
```
Comment (fmichaelobrien) [inline solutions/landing-zone/README.md:183]: may link to one of the solutions - the lz for the full 3 command cycle to fill in the template vars

Comment (fmichaelobrien) [reply solutions/landing-zone/README.md]: never mind - it looks like the following will come in once this branch is synced with main

```
kpt fn render landing-zone
```

File: solutions/landing-zone/README.md [hunk 3]
```diff
@@ -230,93 +234,157 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
...
+        namespace: config-management-system
+      spec:
+        sourceFormat: unstructured
+        sourceType: oci
+        oci:
+          image: northamerica-northeast1-docker.pkg.dev/${PROJECT_ID}/${AR_REPO_NAME}/kustomize-components
+          dir: tenant-a
+          auth: gcpserviceaccount
+          gcpServiceAccountEmail: ${GSA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com
+      EOF
```
Comment (fmichaelobrien) [inline solutions/landing-zone/README.md:397]: I'll need to do a test run of above

File: solutions/landing-zone/README.md [hunk 4]
```diff
@@ -230,93 +236,169 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
...
+      go install github.com/google/go-containerregistry/cmd/crane@latest
+      crane auth login northamerica-northeast1-docker.pkg.dev  -u oauth2accesstoken -p "$(gcloud auth print-access-token)"
+      ```
+
+      #### Render the Configs
+
+      At this point we'll want to update our `setters.yaml` with the appropriate values and render the solution configurations using `kpt`.
+
+      ```
+      kpt fn render
```
Comment (fmichaelobrien) [inline solutions/landing-zone/README.md]: will need to be 
kpt fn render landing-zone

File: solutions/landing-zone/README.md (hunk 2 above)
Comment (cartyc) [reply solutions/landing-zone/README.md]: updated to be `kpt fn render landing-zone`

**Review by fmichaelobrien — APPROVED** [review]
LGTM

File: solutions/landing-zone/README.md (hunk 4 above)
Comment (cartyc) [reply solutions/landing-zone/README.md]: Should be fixed now

File: solutions/landing-zone/README.md (hunk 1 above)
Comment (cartyc) [reply solutions/landing-zone/README.md]: Updated Wording

**Comment by fmichaelobrien** [comment]
LGTM initial passes, Reviewing latest - more after 1630, will apply as much as possible

**Comment by fmichaelobrien** [comment]
LGTM

**Pull request merged**